server/src/test/java/org/elasticsearch/snapshots/SharedClusterSnapshotRestoreIT.java

```diff
@@ -1035,7 +1035,8 @@ public void testUnrestorableFilesDuringRestore() throws Exception {
         final String indexName = "unrestorable-files";
         final int maxRetries = randomIntBetween(1, 10);
 
-        Settings createIndexSettings = Settings.builder().put(SETTING_ALLOCATION_MAX_RETRY.getKey(), maxRetries).build();
+        Settings createIndexSettings = Settings.builder().put(SETTING_ALLOCATION_MAX_RETRY.getKey(), maxRetries)
```

Comment:
It's a little annoying but if we run this test with more than a single shard, then it might be that we get shards that aren't corrupted because they contain no documents now as well as no physical data files because only the `.si` and `segments_N` are uploaded for the empty shards.